### DISCUSSION TITLE
Welcome to wiliot-developer-docs Discussions!

### DISCUSSION THREAD

**Discussion by vyshakhbabji** [vyshakhbabji/wiliot-developer-docs#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by connoralydon** [comment]
Hi All! I'm a developer at Wiliot. The tech is mind boggling and I'm excited to help build world changing tools.
